src/traversal/__tests__/diffRelayQuery_connection-test.js

```diff
@@ -9,6 +9,8 @@
  * @emails oncall+relay
  */
 
+/* eslint-disable indent */
```

Comment:
Again, I'd really like to avoid disables like this. Fixing this file is a bit noisy but if we can time it with making lint fail tests then it's worth it.